sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/healthcare/FhirIOSearchIT.java

```diff
@@ -127,15 +126,15 @@ public void teardown() throws IOException {
     }
   }
 
-  @Ignore("https://github.com/apache/beam/issues/28505")
   @Test
   public void testFhirIOSearch() {
     pipeline.getOptions().as(DirectOptions.class).setBlockOnRun(false);
 
     // Search using the resource type of each written resource and empty search parameters.
     PCollection<FhirSearchParameter<String>> searchConfigs =
         pipeline.apply(
-            Create.of(input).withCoder(FhirSearchParameterCoder.of(StringUtf8Coder.of())));
+            Create.of(input.subList(0, 20))
```

Comment:
Would you mind explaining a little bit how did this change (subList) fix the issue?